samples/drivers/ipm/ipm_esp32/Kconfig.sysbuild

```diff
@@ -1,11 +1,10 @@
 # Copyright 2022 NXP
 #
 # SPDX-License-Identifier: Apache-2.0
-
-source "share/sysbuild/Kconfig"
-
-config IPM_REMOTE_BOARD
-	string
-	default "esp32_devkitc_wrover/esp32/appcpu" if $(BOARD) = "esp32_devkitc_wroom"
-	default "esp32_devkitc_wroom/esp32/appcpu" if $(BOARD) = "esp32_devkitc_wroom"
-	default "esp32s3_devkitm/esp32s3/appcpu" if $(BOARD) = "esp32s3_devkitm"
+#
+#source "share/sysbuild/Kconfig"
+#
+#config IPM_REMOTE_BOARD
+#	string
+#	default "esp32_devkitc_wroom/esp32/appcpu" if $(BOARD) = "esp32_devkitc_wroom"
+#	default "esp32s3_devkitm/esp32s3/appcpu" if $(BOARD) = "esp32s3_devkitm"
```

Comment:
whole file can go?

Reply:
Actually, commenting this was a mistake as I usually do manual builds instead of sysbuild.